Dockerfile

```
@@ -1,7 +1,11 @@
-FROM ocaml/opam:alpine-3.21-ocaml-5.2 AS build
+# syntax=docker/dockerfile:1
+FROM alpine:3.21 AS build
```

```suggestion
FROM alpine:3.22 AS build
```
Sorry, I missed that.